Dockerfile

```diff
@@ -274,20 +275,19 @@ USER root
 RUN curl -o- -L https://yarnpkg.com/install.sh > /usr/local/bin/yarn-installer.sh
 
 # Install node.js
+ENV NVM_VERSION=0.35.3
 USER buildbot
 RUN git clone https://github.com/creationix/nvm.git ~/.nvm && \
     cd ~/.nvm && \
-    git checkout v0.34.0 && \
+    git checkout v"$NVM_VERSION" && \
     cd /
 
 ENV ELM_VERSION=0.19.0-bugfix6
-ENV YARN_VERSION=1.13.0
-
-ENV NETLIFY_BUILD_NODE_VERSION="12.16.1"
+ENV YARN_VERSION=1.22.4
+ENV NODE_VERSION=12.16.2
 
 RUN /bin/bash -c ". ~/.nvm/nvm.sh && \
-         nvm install --no-progress $NETLIFY_BUILD_NODE_VERSION && \
-         nvm install --no-progress 10 && npm install -g sm grunt-cli bower elm@$ELM_VERSION && \
+         nvm install --no-progress $NODE_VERSION && npm install -g sm grunt-cli bower elm@$ELM_VERSION && \
```

Comment:
Are we switching from Node.js `10` to Node.js `12` for the default Node version used by the build image and the `build.command`?

Reply:
Yes, though this will not go into effect until the corresponding PR in bitballoon is merged.